living/scottish_gaelic.md

```diff
@@ -5,41 +5,57 @@ Additional names:
 - 
 
 News:
-- 
-- 
+- https://www.bbc.com/naidheachdan
+- https://www.thenational.scot/topics/gaelic/
+- https://www.gaidhlig.scot/gd/naidheachdan/ (Gaelic Language Board - News Section)
 
 Culture / History:
-- 
-- 
+- https://www.gaidhlig.scot/gd/ (Gaelic Language Board)
+- https://cnag.org/ (The Gaelic Society)
+- https://royalcelticsociety.scot/gd-gb/ (Royal Celtic Society)
+- https://www.taigh-chearsabhagh.org/gd/taigh-tasgaidh-agus-ionad-ealain-taigh-chearsabhagh/ (Taigh Chearsabhagh Museum & Arts Centre)
+- https://www.lismoregaelicheritagecentre.org/?lang=gd (Lismore Gaelic Heritage Centre)
+- https://www.smo.uhi.ac.uk/ (National Centre for Gaelic Language and Culture)
+- https://www.ainmean-aite.scot/gd/ (Gaelic Place-Names of Scotland)
 
 Government:
-- 
-- 
+- https://www.gov.scot/ (Government of Scotland)
+- https://www.parliament.scot/ (Parlament of Scotland)
+- https://www.gov.uk/ (Government of the United Kingdom)
```

Comment:
It would be nice to include the highland subdomain here https://www.highland.gov.uk/ and its Gaelic toolkit https://www.highland.gov.uk/site/gaelic-toolkit/toolkit/index.html